### ISSUE TITLE
Google Material Icons 

### ISSUE THREAD

**Issue by emitanaka** [mitchelloharawild/icons#20]
Been eyeing [this](https://google.github.io/material-design-icons/) for a while. Include in `icon`?

**Comment by mitchelloharawild** [comment]
The key issue for why this isn't included is because the package uses font icons and not SVG icons. This will change along with #19.

I'm also hoping to move away from including the icons in the package, and instead have users provide folders of SVG icons. On the other hand, I would like to keep the ease of including popular icon libraries (such as Font Awesome) easy to use within the package.

My idea was to support these libraries with an installer function, which installs the SVG icons on first usage of that library. This will also make the licensing of the icons within the package a little easier to handle (as we are not re-distributing the icons, simply providing tools to access them).

---

Mostly internal functionality for installing popular libraries:
```r
install_fa()
install_mdi()
install_ai()
install_ii()
```

User-defined custom icons:
```r
my_icon <- new_icon("svgfolder")
```

With exported functionality:
```r
fa()
mdi()
ai()
ii()
my_icon()
```

If the user uses `fa()` without the icons installed, the `install_fa()` will be ran internally.

**Comment by mitchelloharawild** [comment]
Another advantage of not bundling in the icons themselves is that the user can update the fonts at their discretion, and not depend on us updating the package.

**Comment by emitanaka** [comment]
Okay, I haven't thought about this. You bring up a really good point. Extensibility like this where it is not developer dependent is important in the long run.

**Comment by mitchelloharawild** [comment]
Resolved by #31
